distribution/openrct2.d.ts

```diff
@@ -350,6 +351,15 @@ declare global {
         nausea: number;
     }
 
+    interface ActionLocationArgs {
+        readonly x: number;
+        readonly y: number;
+        readonly playerId: number;
```

Comment:
The other network args just uses `player`.

Reply:
player for player Id?